pkg/master/master.go

```diff
@@ -185,10 +185,6 @@ func (c *Config) SkipComplete() completedConfig {
 // Certain config fields must be specified, including:
 //   KubeletClient
 func (c completedConfig) New() (*Master, error) {
-	if c.KubeletClient == nil {
```

Comment:
We don't need KubeletClientConfig to be set?  The comment on 186 needs updating either way though.